myapp/migrations/0006_alter_blog_author_alter_blog_created_at_and_more.py

```diff
@@ -0,0 +1,56 @@
+<<<<<<< HEAD
+# Generated by Django 4.1.5 on 2023-03-01 08:58
+
+import datetime
+=======
+# Generated by Django 4.1.5 on 2023-03-02 03:49
+
+>>>>>>> eaadd19 (Done related_name direct use for template)
```

Comment:
Why this commit details are in migration file. Please be careful when you solve the conflicts. This may throws an error in future.